developing.md

```diff
@@ -11,6 +11,15 @@ yarn install
 yarn package-extension
 ```
 
+Code formatting can be achieved via:
+```bash
+yarn lint --fix
```

Comment:
I would recommend installing these in VSCode.  We are all using VSCode to do development.

![image](https://user-images.githubusercontent.com/1093458/165642370-be10f5c0-e485-4ffd-aa33-4448135bf59e.png)

@christopherswenson do you have an opinion here?

Reply:
Good point, mentioned them here: https://github.com/looker-open-source/malloy/pull/505/commits/9b7c032e99a6e2d2f5126107e2960daeb294cc10